PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,7 +3,12 @@ Checklist:
 <!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->
 
 - [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
-- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
+- [ ] I have read [freeCodeCamp's guidelines for creating a Pull Request](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
```

Comment:
I'm not wild about implying that PR guidelines are something somehow separate from contributing guidelines. PR guidelines should be a subset of contributing guidelines.

In general, it is rare that we have issues, so I'm not sure that making this template more complicated is a good answer to 'fix' a rare problem. We should target the 95% scenarios with our templates, not the 5% edge cases, IMHO.

Reply:
I don't think this implies separation, as much as the request for a descriptive title (which is already present) implies separatation from the contribution guidelines. What I'm looking to achieve here is effective sign posting of a subset of the guidelines that applies to all Pull Requests, and hence I believe is worthwhile highlighting. As simply stating `don't do 'x'` is vastly less helpful when compared to:

1. Pointing towards the informative pull request page
2. Giving clear examples of what you _should_ do, which is a noticeably shorter list than what you _shouldn't_.

Even if we take your suggested numbers as reality, isn't adding a single line to a template to remove the isues to the ~0% realm still a good thing? Afterall, what is the increase of complexity for a new user if instead of being clearly pointed to the relevant resource, they instead have to read through the contribution guidelines until they find it? I would wager this complexity is greater than a single line.

Reply:
> Even if we take your suggested numbers as reality, isn't adding a single line to a template to remove the isues to the ~0% realm still a good thing? 

I honestly don't think so. I'm all for reviewing the guidelines periodically, but I think that workflows and processes should be kept as simple as possible. More is not always better.

I understand that it is frustrating that you received a ban, but those bans are rare, yours was reversed, and the system worked.

How many of the closed PRs actually are actually rejected for failing to meet our contributing guidelines? Very few. We have a reoccurring issue with people making a single line change in a PR that isn't needed because they are following a YouTube tutorial, but those are automatically closed with no moderation activity. We have some spam during Hacktoberfest, but that's just unavoidable for a large repository. Other than that, things are pretty good. It's rare that people run afowl of the contributing guidelines, and rarer still that it results in a ban.

I see reviewing documentation and guidelines as always good. I see adding complexity and process as rarely good. I think this PR is looking at the wrong place.

Reply:
I agree it should be kept simple, which is why I thought an explicit call out to the applicable section of the guidelines would be a good improvement. Similar to how there are explicit callouts for information in the creation of issues, I think PRs could benefit from this. This isn't adding any process that isn't already implicit with the link to the contribution guidelines, just a more explicit highlighting of it.  

Based off a previous discussion with another moderator it seemed as if this issue (new joiners not finding the relevant area of the docs) was prevalent, but maybe this is incorrect. I would like to point out that just because other people aren't getting banned as I did, doesn't mean there isn't room for improvement.  

Feel free to close out the PR if you still feel it's not an improvement.